tests/hypothesis_strategies/priors.py

```diff
@@ -0,0 +1,11 @@
+"""Hypothesis strategies for priors."""
```

Comment:
@AdrianSosic here is the error that we discussed earlier, screenshot taken from the logs: 
![image](https://github.com/emdgroup/baybe/assets/84016733/050dfabb-caf5-4c42-be59-868f8eeb7d57)

Reply:
Problem identified, as mentioned via chat. `kw_only=True` missing in the `lengthscale_prior` field of the `Kernel` base class